tools/src/listProtos.ts

```diff
@@ -22,8 +22,8 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as walk from 'walkdir';
 
-const googleProtoFilesDir = path.join(__dirname, '..', '..', 'protos');
```

Comment:
@alexander-fenster, now that this is it's own module, this seems like a pretty fragile implementation. I don't want to change it now, since I think we should leave the code changes as minimal as possible, but I think in the future it would be good to make these paths a bit more robust. WDYT?